openfasoc/generators/cryo-gen/tools/cryo-gen.py

```diff
@@ -299,14 +302,27 @@
 p = sp.Popen(["yum", "install", "-y", "libXaw"])
```

Comment:
@Jerry-lqx this is not portable code. Basically, any machine that does not have yum will not run this line. Also does not work without sudo depending on the machine's settings with the package manager. You need to remove completely and update dependencies in readme so user's can install themselves. Making this portable is possible with python but very difficult.